mvt/android/cli.py

```diff
@@ -54,9 +55,12 @@ def download_apks(all_apks, virustotal, koodous, all_checks, output, from_file):
         if from_file:
             download = DownloadAPKs.from_json(from_file)
         else:
-            if not output:
-                log.critical("You need to specify an output folder (with --output, -o) when extracting APKs from a device")
-                sys.exit(-1)
+            try:
```

Comment:
I think you missed the case where output is None, as you removed the `if not output`. (applies to all changed you made here, maybe you can wrap them in a function as well? Just to keep repetition down)